components/edit-func.js

```diff
@@ -92,7 +93,7 @@ ControlButton.propTypes = {
   isDisabled: PropTypes.bool,
 };
 
-const EditJson = ({ text, onChange, enabled = true, errors = [] }) => {
+const EditJson = ({ text = "", onChange, enabled = true, errors = [] }) => {
```

Comment:
I was just noticing a quick warning that shows up in the web console logs that when the you first click on the edit button it will say that text is undefined but should be defined, so I was trying to force it. Since it's not actually a problem, because it loads properly almost immediately afterward, I just wanted to make the warning go away. Is the warning okay to just exist?

Reply:
That tells me `EditJson` is getting called with null or undefined text, which should get fixed, or else the propTypes have no purpose. Unless an empty string can trigger a propTypes warning, which I don’t _think_ happens.